setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open("requirements.txt", "r") as fh:
+    requirements = fh.read()
+
+setuptools.setup(
+    name="subtest",
+    version="0.0.0.6",
+    author="SubstraFoundation",
+    author_email="hello@substra.org",
+    description="A distributed learning contributivity package",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/SubstraFoundation/distributed-learning-contributivity",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Operating System :: OS Independent",
+    ],
+    install_requires=requirements,
+    python_requires='>=3.6',
+)
```

Comment:
Please add the `license` and `keyword` fields & we'll have to elaborate the classifiers

Reply:
I'm note sure yet what yo put for keyword so I used placeholders

Reply:
Cool! I'd add "machine learning", "contributivity", "multipartner". Any suggestion @RomainGoussault.